pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -957,13 +958,6 @@ func (service *SecurityPolicyService) buildPeerTags(obj *v1alpha1.SecurityPolicy
 		groupTypeTag = String(common.TagValueGroupSource)
 		peers = &rule.Sources
 	}
-
-	// TODO: abstract sort func for both peers and targets
-	sort.Slice(*peers, func(i, j int) bool {
-		k1, _ := json.Marshal((*peers)[i])
-		k2, _ := json.Marshal((*peers)[j])
-		return string(k1) < string(k2)
-	})
```

Comment:
I am confused about the use case of this tag, do we really have a consumer on the hash value of the selectors? If no, can we remove it? If yes, does the consumer use `sort` on the selectors to calculate the tag, then this change is possibly break it.

Reply:
As we discussed, we don't have consumer for these hash value by `sort`. So, it won't break the logic.
In addition, the `sort` function will change rule selector order, it will change rule hash value after `sort`.
but we use this rule hash in our ID and name, we need to keep it consistent. That is why we removed it.